docs/community/100-days-of-oss.md

```diff
@@ -3,79 +3,100 @@ id: 100-days-of-oss
+This is a great time to show your technical growth. Here are some paths you can take:
+
+- Update or write documentation to improve clarity and usability.
+- Maintain a project: review pull requests, triage issues, and respond to questions.
+- Submit pull requests to contribute code changes.
+
+### Tracking Your Progress
+
+To keep track of your progress, post on social media, your blog, or any other platform you prefer with:
+
+- progress you made,
+- the day of the challenge indicated by 'D',
+- the hashtag #100DaysOfOSS.
+
+For example, if you're on day one, you could say, "Today, I reviewed the documentation for the [OpenSauced/App](https://github.com/open-sauced/app) repository. D1 #100daysOfOSS." Then, on day two, you would continue with D2, and so on.
 
 Here's what you can do if you're ready to join this challenge:
-- [Sign up](https://docs.google.com/forms/d/e/1FAIpQLScoq6ORpkzUg3Cqxd3EeNnMP6HIJ3uemPhSHs8sYZALmbpeVg/viewform?usp=sharing) with OpenSauced to receive a coupon code for 12 months of free access to all OpenSauced's paid features, and for the chance to opt in to receive updates, event invites, and resources to help you succeed.
-- [Tweet out your commitment today](https://twitter.com/intent/tweet?text=I%27m%20joining%20the%20%23100DaysOfOSS%20challenge%20with%20%40saucedopen%20to%20grow%20my%20skills%2C%20gain%20real-world%20experience%20in%20open%20source%2C%20and%20to%20make%20a%20positive%20impact%20on%20the%20OSS%20Community!) or share on your platform of choice!
+
+- [Sign up](https://docs.google.com/forms/d/e/1FAIpQLScoq6ORpkzUg3Cqxd3EeNnMP6HIJ3uemPhSHs8sYZALmbpeVg/viewform?usp=sharing) with OpenSauced to receive a coupon code for 12 months of free access to all of OpenSauced's paid features and for the chance to opt in to receive updates, event invites, and resources to help you succeed.
+- [Post your commitment today on X](https://twitter.com/intent/tweet?text=I%27m%20joining%20the%20%23100DaysOfOSS%20challenge%20with%20%40saucedopen%20to%20grow%20my%20skills%2C%20gain%20real-world%20experience%20in%20open%20source%2C%20and%20to%20make%20a%20positive%20impact%20on%20the%20OSS%20Community!) or share on your platform of choice!
 
 ## The Official Kickoff
+
 We're starting 100 days on July 23rd to the end of [Hacktoberfest](https://hacktoberfest.com/), a month-long celebration of open source contributions. We'll provide continuous support, daily inspirational tweets, and engaging events to help you stay motivated and make progress.
 
-Don't worry if you're unable to start on the same day as everyone else. The #100DaysOfOSS challenge is flexible, and you can join in whenever you're ready. Just jump in at any point and begin with day one of your personal challenge. 
+Don't worry if you're unable to start on the same day as everyone else. The #100DaysOfOSS challenge is flexible; you can join in whenever you're ready. Just jump in at any point and begin with day one of your personal challenge.
```

Comment:
In my opinion, we still can keep the `#100DaysOfOSS` page to motivate and give opportunity for folks doing it anytime like `#100DaysOfCode`.

However, we might want to remove some sections that are not applicable anymore like this one.